packages/connect-explorer/src-webextension/background/serviceWorker.ts

```diff
@@ -1,3 +1,7 @@
 /// <reference lib="webworker" />
 
-importScripts('vendor/trezor-connect-webextension.js');
+import TrezorConnect, { DEVICE_EVENT } from '@trezor/connect-webextension';
+
+TrezorConnect.on(DEVICE_EVENT, (event: any) => {
+    console.log('DEVICE_EVENT', event);
+});
```

Comment:
If I understood well, we are adding it here just so Webpack does not get rid of the previous import.

Maybe adding a comment here about why we add here this log could be useful. Since we are not doing it for the obvious reason which is having a log but for the reason that we do not want Webpack to do tree-shaking with it.

:question:

Reply:
I am not sure this is the case. Isn't it only to test that both named and default exports work as expected?

Reply:
Yes it's actually because Webpack was getting rid of it, at least with our config

Reply:
What about adding some comments about why this log is here?
Not only for other people looking at this but for our future us, so we don't forget about the reason.